src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -13,7 +13,7 @@
             </div>
         </div>
         <div class="row mt-2">
-            <div v-if="selectedTab==='map'" id="choropleth-container" class="col-md-12">
+            <div v-if="selectedTab === modelOutputTabs.Map" id="choropleth-container" class="col-md-12">
```

Comment:
I updated this to an enum too as did some switching on it in a later action and thought the enum would be a bit of a tidier way to do that